bin/zenossdbpack

```diff
@@ -101,6 +86,8 @@ if __name__ == "__main__":
         "-d", "--days", dest="days", type=float, default=0.5, # 12 hours
         help="Days to keep unreferenced objects (default: %(default)s). "
         "NOTE: To help prevent PKEs, zenossdbpack will not accept --days below 0.5")
+    # -e option is no longer used. Left for compatibility purposes.
+    # Many customers have cron jobs passing -e session
     parser.add_argument(
         "-e", "--exclude-db", dest="excludeDB",
         choices=['session', 'zodb'],
```

Comment:
I think you should change help text for this argument to say it has no effect - just what you've done in comment above for developers